test-utils/runtime/src/lib.rs

```diff
@@ -1360,13 +811,184 @@ fn test_witness(proof: StorageProof, root: crate::Hash) {
 	assert!(ext.storage_root(Default::default()).as_slice() != &root[..]);
 }
 
+/// Some tests require the hashed keys of the storage. As the values of hashed keys are not trivial
+/// to guess, this small module provides the values of the keys, and the code which is required to
+/// generate the keys.
+#[cfg(feature = "std")]
+pub mod storage_key_generator {
+	use super::*;
+	use sp_core::Pair;
+	use sp_keyring::AccountKeyring;
+
+	fn concat_hashes(input: &Vec<&[u8]>) -> String {
+		// sp_core::hashing::twox_128(input[0]).encode_hex::<String>()
+		input.iter().map(|s| sp_core::hashing::twox_128(s)).map(hex::encode).collect()
+	}
+
+	fn twox_64_concat(x: &[u8]) -> Vec<u8> {
+		sp_core::hashing::twox_64(x).iter().chain(x.iter()).cloned().collect::<Vec<_>>()
+	}
+
+	/// Generate the hashed keys from the raw literals
+	pub fn generate_expected_keys() -> Vec<String> {
```

Comment:
The name should be better, it is really hard to know what this is about.

Reply:
Improved in: https://github.com/paritytech/substrate/pull/13737/commits/22069a061215baa3a4c7d3ba7d163fb06c807a06